dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -23,7 +23,7 @@ export interface WindowGuardianConfig {
 		googletag: string;
 	};
 	switches: { [key: string]: boolean };
-	tests?: { [key: string]: string };
+	tests: ServerSideTests;
```

Comment:
This was optional before, are we now sure it will always be provided?

Reply:
Yes, from the look of https://www.theguardian.com/uk-news/2021/oct/26/woman-cleared-obstructing-met-police-partner-shot-stun-gun.json?dcr (assuming you’re not in any test)

![abTest empty object](https://user-images.githubusercontent.com/76776/145434404-4ad27762-7430-4ce9-a007-3aaa546a7d69.png)

Reply:
Nice 👍